frontends/main/src/app-pages/DashboardPage/CoursewareDisplay/transform.ts

```diff
@@ -113,7 +132,7 @@ const sortDashboardCourses = (
   program: DashboardProgram,
   courses: DashboardCourse[],
 ) => {
-  return courses.sort((a, b) => {
+  return [...courses].sort((a, b) => {
```

Comment:
Avoid mutating the original array. Mutating the array caused an issue in the jest tests (basically, made assertions harder).